shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -340,15 +346,21 @@ \subsection{New Epoch Transition}
       e = e_\ell + 1
       &
       \var{ru} \neq \Nothing
-      \\~\\
+      \\
+      (\wcard,~\wcard,~\wcard,~\wcard,~,\var{irwd'})\leteq\var{ru}
```

Comment:
Typo: There is an extra "," just in front of `irwd'`.

Reply:
thanks!